.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: serv ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  compliance:
+    name: RISC-V Compliance Test
+    runs-on: ubuntu-20.04
+    env:
+      SERVANT_VERSION: 1.0.2
```

Comment:
@olofk is there any way to run a command to get the latest version of sevant? Maybe through fusesoc?